code_examples/sdk_examples/src/core_features/getting_started/06_verify_credential.ts

```diff
@@ -1,17 +1,15 @@
 import * as Kilt from '@kiltprotocol/sdk-js'
+import { VerifiableCredential } from '@kiltprotocol/credentials/lib/cjs/V1/types'
```

Comment:
this is not a valid import, you're probably looking for `import { types } from '@kiltprotocol/credentials'` & then `types.VerifiableCredential`